app/assets/javascripts/ga_cross_domain_form_tracking.js

```diff
@@ -0,0 +1,17 @@
+function attachCrossDomainTrackerToInput(form, trackers) {
```

Comment:
We should probably have a spec file for this code. Should be a case of having a chunk of form markup, running the code on it and checking the action attribute has been updated. Might need some cleverness to test the right ga parameter has been appended.

Reply:
humm, I've not done a lot of mocking and stubbing in Jasmine, would need to go read up.

Would be happy to pair through this if you've got an idea of how this would work?